scripts/check_type_hints.py

```
@@ -194,13 +225,98 @@ def add_future_import(file_path: Path) -> bool:
+        python_files.extend(src_dir.rglob("*.py"))
+
+    if include_tests:
+        tests_dir = Path("tests")
+        if tests_dir.exists():
+            python_files.extend(tests_dir.rglob("*.py"))
+
+    return python_files
+
+
+def _process_file(
+    file_path: Path, fix: bool
```

Ruff (0.14.7)

Boolean-typed positional argument in function definition

(FBT001)